frontend/vue/components/constants/megaMenuLinks.ts

```diff
@@ -484,45 +484,52 @@ const QUANTUM_COMPUTING_LABS : MegaDropdownMenuGroup = {
       }
     },
     {
-      label: 'Lab 2. Quantum Measurement',
-      url: `${baseUrl}${pathLabs}/lab-2-quantum-measurements`,
+      label: 'Lab 2. Single Quibit Gates',
+      url: `${baseUrl}${pathLabs}/lab-2-single-quibit-gates`,
```

Comment:
Typo, fixable after qiskit-community/platypus#220 is merged:

```suggestion
      url: `${baseUrl}${pathLabs}/lab-2-single-qubit-gates`,
```

Reply:
@eddybrando thanks again. good catch!